### PR TITLE
Run integration tests in parallel

### PR DESCRIPTION
**Pull request by mdharamadas1**
Fix for #1196 
Also, added README for tests.

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Automated Integration Testing
+
+## Approach
+
+Automated tests are designed and developed using [Robot Framework](https://github.com/robotframework/robotframework) and are executed using pybot or pabot.
+These tests are a mix of backend and browser UI tests and are executed in parallel within [Drone](https://github.com/drone/drone) container on CI.
+Browser tests are run using [Docker Selenium Grid](https://github.com/SeleniumHQ/docker-selenium) with headless Firefox nodes which are started per build basis as Drone services.
+
+Common OVA appliance is installed as part of the CI pipeline, before the integration test step, which is shared among all of the tests. It is cleaned up later once the test run is completed.
+All the robot logs and screenshots are published on [Google storage](https://console.cloud.google.com/storage/browser/vic-ci-logs?project=eminent-nation-87317).
```
Comment (andrewtchin) [inline tests/README.md]: Is that bucket for engine too? If so, should these be a separate bucket?

Comment (mdharamadas1) [reply tests/README.md]: Yeah, I'm fine with either buckets. If you feel they should be separate, let me know I will create one.

Comment (andrewtchin) [reply tests/README.md]: As long as there is a link to the correct log bundle in the build output and the log bundle file name shows what repo and build number it is for I guess it's ok in the same

Comment (mdharamadas1) [reply tests/README.md]: Yeah, link is displayed in the logs and looks like below:
`https://console.cloud.google.com/m/cloudstorage/b/vic-ci-logs/o/ova_integration_logs_2885_04bad15bce6f905bcdb556e766133e1d622791ea.zip?authuser=1`

**Pull request merged**